diff -Naur a/src/Makefile b/src/Makefile
--- a/src/Makefile	2019-06-25 16:07:38.175291868 +0100
+++ b/src/Makefile	2019-06-25 16:08:10.420291970 +0100
@@ -36,7 +36,7 @@
 #HAVE_NSS = -DHAVE_NSS `pkg-config --cflags nss`
 
 # Change sm_10 to sm_20 if you have Fermi support (400 series or later)
-NVCC_FLAGS = -c -Xptxas -v -arch sm_10
+NVCC_FLAGS = -c -Xptxas -v -arch sm_30
 CUDAPATH = /usr/local/cuda/lib
 CUDA64PATH = /usr/local/cuda/lib64
 
